src/RA_LeaderboardPopup.cpp

```diff
@@ -171,12 +169,14 @@ _Use_decl_annotations_ void LeaderboardPopup::Render(ra::ui::drawing::ISurface&
             if (!pConfiguration.IsFeatureEnabled(ra::services::Feature::LeaderboardCounters))
                 break;
 
-            const auto& pSurfaceFactory = ra::services::ServiceLocator::Get<ra::ui::drawing::ISurfaceFactory>();
+            using default_isurfacefactory_t = ra::ui::drawing::ISurfaceFactory;
```

Comment:
this alias is undesirable. it's barely shorted, and abstracts the usage in the lookup.

Reply:
Sorry, I must have forgotten to get rid of it. I was originally trying to make the class a class template that accepted any integral type for the dimensions.